### PR TITLE
Update adobe stock integration configuration instructions

### PR DESCRIPTION
**Pull request by sivaschenko**
## Purpose of this pull request

Updated adobe stock integration configuration instructions based on the recent console.adobe.io change

Closes https://github.com/magento/adobe-stock-integration/issues/1322

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

https://docs.magento.com/user-guide/cms/adobe-stock.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cms/adobe-stock.md [hunk 1]
```diff
@@ -32,13 +32,20 @@ then [configure the Adobe Stock Integration in your Magento Admin][configure].
...
 
-   - The default redirect URI is in the form `${HOST}/${ADMIN_URI}/adobe_ims/oauth/callback/.*`, such as `https://store.myshop.com/admin_hgkq1l/adobe_ims/oauth/callback/.*`.
+   - The default redirect URI is in the form `${HOST}/${ADMIN_URI}/adobe_ims/oauth/callback/`, such as `https://store.myshop.com/admin_hgkq1l/adobe_ims/oauth/callback/`.
 
         where:
 
          - `${HOST}` is your Magento fully qualified domain name (i.e., `https://store.myshop.com`).
          - `${ADMIN_URI}` is your Magento Admin URI (such as `admin_hgkq1l`), which can be retrieved by running `magento info:adminuri`.
 
+1. Specify the **Redirect URI pattern**. This is the same as your redirect URI with two differences:
```
Comment (leslietilling) [inline src/cms/adobe-stock.md:42]:
```suggestion
1. Specify the **Redirect URI pattern**, which is the same as your redirect URI with two differences:
```

File: src/cms/adobe-stock.md [hunk 2]
```diff
@@ -32,13 +32,20 @@ then [configure the Adobe Stock Integration in your Magento Admin][configure].
...
+
+   - Any periods (`.`) must be escaped with two backslashes (`\\`)
```
Comment (leslietilling) [inline src/cms/adobe-stock.md:44]:
```suggestion
   - Any periods (`.`) must be escaped with two backslashes (`\\`).
```

File: src/cms/adobe-stock.md [hunk 3]
```diff
@@ -32,13 +32,20 @@ then [configure the Adobe Stock Integration in your Magento Admin][configure].
...
+   - Add `.*` to the end of the pattern
```
Comment (leslietilling) [inline src/cms/adobe-stock.md:45]:
```suggestion
   - Add `.*` to the end of the pattern.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**